Storage/api_types.h

```diff
@@ -34,10 +34,14 @@ namespace current {
 namespace storage {
 namespace rest {
 
+// Top-level URL prefixes for Storage REST.
 const std::string kRESTfulDataURLComponent = "data";
 const std::string kRESTfulSchemaURLComponent = "schema";
+const std::string kRESTfulCQRSCommandURLComponent = "command";
+const std::string kRESTfulCQRSQueryURLComponent = "query";
```

Comment:
I vote for the second level here: "cqrs/command", "cqrs/query".
Top-level "query" may be used for the future generic query engine.